ui/text_displayer.js

```diff
@@ -361,4 +365,75 @@ shaka.ui.TextDisplayer = class {
       panelStyle.height = cue.size + '%';
     }
   }
+
+  /**
+   * Returns info about provided lengthValue
+   * @example 100px => { value: 100, unit: 'px' }
+   * @param {?string} lengthValue
+   *
+   * @return {?{ value: number, unit: string }}
+   * @private
+   */
+  static getLengthValueInfo_(lengthValue) {
+    const matches = new RegExp(/(\d*\.?\d+)([a-z]+|%+)/).exec(lengthValue);
+
+    if (!matches ) {
+      return null;
+    }
+
+    return {
+      value: Number(matches[1]),
+      unit: matches[2],
+    };
+  }
+
+  /**
+   * Converts length value
```

Comment:
"Converts length value" returning "string" is not enough information for the reader.  What format is the string?  What sort of conversion?

Reply:
the description was updated,
please take a look if I need to add more details